doc/man7/provider-base.pod

```diff
@@ -332,6 +344,72 @@ pointing at the string "foo,bar"
 For more information on handling parameters, see L<OSSL_PARAM(3)> as
 L<OSSL_PARAM_int(3)>.
 
+=head1 CAPABILITIES
+
+Capabilties describe some of the services that a provider can offer.
```

Comment:
Typo: Capabilties

Reply:
As previously mentioned to @slontis, this is entirely the fault of my keyboard:

> I think there's a problem with my keyboard. It can't seem to spell that word.

Reply:
singer instead of signer is one of my favorites.